common/lib/xmodule/xmodule/js/spec/video/video_full_screen_spec.js

```diff
@@ -0,0 +1,102 @@
+(function () {
+    'use strict';
+    describe('VideoFullScreen', function () {
+        var state, oldOTBD;
+
+        beforeEach(function () {
+            oldOTBD = window.onTouchBasedDevice;
+            window.onTouchBasedDevice = jasmine
+                .createSpy('onTouchBasedDevice').andReturn(null);
+        });
+
+        afterEach(function () {
+            $('source').remove();
+            state.storage.clear();
+            state.videoPlayer.destroy();
+            window.onTouchBasedDevice = oldOTBD;
+        });
+
+        describe('constructor', function () {
+            beforeEach(function () {
+                state = jasmine.initializePlayer();
+            });
+
+            it('render the fullscreen control', function () {
+                expect($('.add-fullscreen')).toExist();
+                expect(state.videoFullScreen.fullScreenState).toBe(false);
+            });
+
+            it('add ARIA attributes to fullscreen control', function () {
+                var fullScreenControl = $('.add-fullscreen');
+
+                expect(fullScreenControl).toHaveAttrs({
+                    'role': 'button',
+                    'title': 'Fill browser',
+                    'aria-disabled': 'false'
+                });
+            });
+
+            it('event handler to toggle fullscreen mode', function () {
```

Comment:
Some of the test names in this file do not follow the "it (does something)" convention e.g. this could be: "correctly triggers the event handlers for fullscreen mode".